### PR TITLE
Request code review feature

### PR DESCRIPTION
**Pull request by danMateer**
This doc details the **Request code review** beta feature that was GA'd last week.

@martijnrusschen I couldn't get this up and running on `localhost:8000`; Gatsby kept yelling at me:
> There's not a page or function yet at `/organizations.html`

So I modeled this after the other PR #1000 for ineligible findings that also went out from Hack Week. It's a pretty basic content update, though.

If you can sanity check that this is rendering okay I'd be forever grateful.

### CODE REVIEW

**Comment by danMateer** [comment]
@jobertabma @tylermann one of y'all mind giving this a once-over?

File: docs/organizations/request-code-review.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+In addition to confirming a vulenerability is has been fixed by requesting a [Retest](retest.html), you can also request a code review to verify any source code updates adequately address the reported vulneratiblity. These are conducted by the HackerOne Code Review Community: a group of rigorously-vetted, specialized software engineers and security experts HackerOne created as part of the [acquisition of PullRequest.com](https://www.hackerone.com/press-release/hackerone-acquires-pullrequest-power-developer-first-security-testing-solutions). You can learn more about this specialized cohort of the HackerOne Community [here](https://www.pullrequest.com/reviewers/).
+
+The **Request code review** feature allows you to send a limited code snippet patch to be reviewed as generated by `git diff`.
+
+> **Note:** This feature is currently in open beta.
+
+### How it Works  
+To have request a patch code review:
```
Comment (danMateer) [inline docs/organizations/request-code-review.md:14]:
```suggestion
To request a patch code review:
```

File: docs/organizations/request-code-review.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+How much of the source code is exposed to reviewers? | Only what is contained within the diff snippet. These contain whatever lines of code have been changed and, where applicable, a few lines of code above and below the changed lines to provide additional context for the impacted lines of code.
+If a reviewer finds a problem with the proposed fix, can I submit another diff snippet with updated code? | Yes. We encourage it!
+Can I respond to the reviewer or ask questions? | Yes. Simply respond with an [Internal](/report-components.html#timeline) comment.
+What programming languages, frameworks, libraries and platforms do you support? | All types of code are supported. This includes Infrastructure as Code (IaS) and Configuration as Code (CaC). It does help our community if you expand the Context to include version(s) of a programming language or framework if relevant to determine correctness. Our systems determine the type of code submitted by the file types included as part of the diff snippet and assign to experts in our community accordingly.
```
Comment (danMateer) [inline docs/organizations/request-code-review.md:47]:
```suggestion
What programming languages, frameworks, libraries and platforms do you support? | All types of code are supported. This includes Infrastructure as Code (IaS) and Configuration as Code (CaC). It does help our community if you expand the Context to include version(s) of a programming language or framework if relevant to determine correctness. Our systems detect the type of code submitted by the file types included as part of the diff snippet and assign to experts in our community accordingly.
```

File: docs/organizations/request-code-review.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Request Code Review (BETA)"
+path: "/organizations/request-code-review.html"
+id: "organizations/request-code-review"
+---
+
+In addition to confirming a vulenerability is has been fixed by requesting a [Retest](retest.html), you can also request a code review to verify any source code updates adequately address the reported vulneratiblity. These are conducted by the HackerOne Code Review Community: a group of rigorously-vetted, specialized software engineers and security experts HackerOne created as part of the [acquisition of PullRequest.com](https://www.hackerone.com/press-release/hackerone-acquires-pullrequest-power-developer-first-security-testing-solutions). You can learn more about this specialized cohort of the HackerOne Community [here](https://www.pullrequest.com/reviewers/).
```
Comment (danMateer) [inline docs/organizations/request-code-review.md:7]:
```suggestion
In addition to confirming a vulnerability is has been fixed by requesting a [Retest](retest.html), you can also request a code review to verify any source code updates adequately address the reported vulnerability. These are conducted by the HackerOne Code Review Community: a group of rigorously-vetted, specialized software engineers and security experts HackerOne created as part of the [acquisition of PullRequest.com](https://www.hackerone.com/press-release/hackerone-acquires-pullrequest-power-developer-first-security-testing-solutions). You can learn more about this specialized cohort of the HackerOne Community [here](https://www.pullrequest.com/reviewers/).
```

Comment (danMateer) [reply docs/organizations/request-code-review.md]: I swear I can spell.

File: docs/organizations/request-code-review.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Request Code Review (BETA)"
+path: "/organizations/request-code-review.html"
+id: "organizations/request-code-review"
+---
+
+In addition to confirming a vulnerability is has been fixed by requesting a [Retest](retest.html), you can also request a code review to verify any source code updates adequately address the reported vulnerability. These are conducted by the HackerOne Code Review Community: a group of rigorously-vetted, specialized software engineers and security experts HackerOne created as part of the [acquisition of PullRequest.com](https://www.hackerone.com/press-release/hackerone-acquires-pullrequest-power-developer-first-security-testing-solutions). You can learn more about this specialized cohort of the HackerOne Community [here](https://www.pullrequest.com/reviewers/).
```
Comment (danMateer) [inline docs/organizations/request-code-review.md:7]:
```suggestion
In addition to confirming a vulnerability has been fixed by requesting a [Retest](retest.html), you can also request a code review to verify any source code updates adequately address the reported vulnerability. These are conducted by the HackerOne Code Review Community: a group of rigorously-vetted, specialized software engineers and security experts HackerOne created as part of the [acquisition of PullRequest.com](https://www.hackerone.com/press-release/hackerone-acquires-pullrequest-power-developer-first-security-testing-solutions). You can learn more about this specialized cohort of the HackerOne Community [here](https://www.pullrequest.com/reviewers/).
```

**Review by martijnrusschen — APPROVED** [review]
Reviewed and checked out on a local branch and looks like it's working fine.

**Pull request merged**